### PR TITLE
Map chunk and chunk statuses

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -25,6 +25,12 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_23350 getJumpVelocityMultiplier ()F
 	METHOD method_27839 is (Lnet/minecraft/class_2248;)Z
 		ARG 1 block
+	METHOD method_30094 replaceBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;I)V
```
Comment (liach) [inline mappings/net/minecraft/block/Block.mapping:28]: `replaced`?

File: mappings/net/minecraft/world/chunk/ProtoChunk.mapping [hunk 2]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2839 net/minecraft/world/chunk/ProtoChunk
+	COMMENT
```
Comment (liach) [inline mappings/net/minecraft/world/chunk/ProtoChunk.mapping]:
```suggestion
```

Comment (i509VCB) [reply mappings/net/minecraft/world/chunk/ProtoChunk.mapping]: Hmm, I thought I purged all the orphaned comments

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/block/Block.mapping:28]: If this is called when this block is replaced, I'd suggest
```suggestion
	METHOD method_30094 replaced (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;I)V
```

Comment (i509VCB) [reply mappings/net/minecraft/block/Block.mapping:28]: This is the exact same name as the other method that also passes the maximum depth. Unless we want to change that one also (not an issue tbf)

Comment (liach) [reply mappings/net/minecraft/block/Block.mapping:28]: Hmm, how often is this method overridden by mods? Need to weigh the benefit and the cost.

But replaceBlock sounds like this block is replacing other block than this block is getting replaced.

**Comment by i509VCB** [comment]
I think I have to make sure this doesn't end up the same as #1493

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (i509VCB) [reply mappings/net/minecraft/block/Block.mapping:28]: This is a static method

File: mappings/net/minecraft/world/ChunkRegion.mapping [hunk 3]
```diff
@@ -9,14 +9,14 @@ CLASS net/minecraft/class_3233 net/minecraft/world/ChunkRegion
 	FIELD field_14094 fluidTickScheduler Lnet/minecraft/class_1951;
 	FIELD field_14096 dimension Lnet/minecraft/class_2874;
 	FIELD field_14097 levelProperties Lnet/minecraft/class_5217;
-	FIELD field_14098 chunks Ljava/util/List;
+	FIELD field_14098 surroundingChunks Ljava/util/List;
```
Comment (Gegy) [inline mappings/net/minecraft/world/ChunkRegion.mapping:12]: `surroundingChunks` in this context feels like it implies only the edge chunks. I think `chunks` might be descriptive enough?

Comment (i509VCB) [reply mappings/net/minecraft/world/ChunkRegion.mapping:12]: maybe neighboring chunks?

Comment (Gegy) [reply mappings/net/minecraft/world/ChunkRegion.mapping:12]: In the context of `ChunkStatus` that makes sense, but I think `ChunkRegion` is used more generically. It's just all the chunks contained within the area. If `chunks` doesn't work, maybe `containedChunks`/`includedChunks` or something along those lines?

Comment (i509VCB) [reply mappings/net/minecraft/world/ChunkRegion.mapping:12]: On further thought, `chunks` should be fine

**Comment by i509VCB** [comment]
Looks like I have to reopen this because of detached branch

**Comment by liach** [comment]
Yeah just create a new fork and push this branch to the new fork's remote and open a new pr

**Comment by i509VCB** [comment]
Reattached at #1511

**Pull request closed without merging**